### PR TITLE
zh-translation: docs/setup/install/operator/index.md

### PR DESCRIPTION
**Pull request by TianTianBigWang**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/setup/install/operator/index.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ Istio [operator](https://kubernetes.io/zh/docs/concepts/extend-kubernetes/operat
 管理安装。
 这样做还能缓解管理不同 Istio 版本的负担。
 你只需简单的更新 operator {{<gloss CRDs>}}自定义资源（CR）{{</gloss>}}即可，
-operator 控制器将为你应用相应的配置更改。
+operator 控制器将为你应用更改的相应配置。
 
-当你用 [istioctl install 命令](/zh/docs/setup/install/istioctl)安装 Istio 时，
+当你用 [istioctl install 说明](/zh/docs/setup/install/istioctl)安装 Istio 时，
 底层使用的是和 operator 安装相同的
 [`IstioOperator` API](/zh/docs/reference/config/istio.operator.v1alpha1/)。
```
Comment (orangegzx) [inline content/zh/docs/setup/install/operator/index.md:19]:
```suggestion
当你用 [istioctl Install 说明](/zh/docs/setup/install/istioctl)安装 Istio 时，底层使用的是和 Operator 安装相同的
[`IstioOperator` API](/zh/docs/reference/config/istio.operator.v1alpha1/)。
```

Comment (orangegzx) [reply content/zh/docs/setup/install/operator/index.md:19]: 中文不应该换行

File: content/zh/docs/setup/install/operator/index.md [hunk 2]
```diff
@@ -57,7 +57,7 @@ operator 控制器将为你应用相应的配置更改。
     更多详细信息，请参阅 [`istioctl operator init` 命令参考](/zh/docs/reference/commands/istioctl/#istioctl-operator-init)。
 
     {{< tip >}}
-    您也可以使用 Helm 部署 operator：
+    你也可以使用 Helm 部署 operator：
```
Comment (orangegzx) [inline content/zh/docs/setup/install/operator/index.md:60]:
```suggestion
    你也可以使用 Helm 部署 Operator：
```

File: content/zh/docs/setup/install/operator/index.md [hunk 3]
```diff
@@ -89,6 +89,23 @@ EOF
...
+...
+spec:
+  profile: demo
+  values:
+    global:
+      istioNamespace: istio-namespace1
+{{< /text >}}
+
 {{< tip >}}
 Istio operator 控制器在创建 `IstioOperator` 资源的90秒内启动 Istio 的安装。
```
Comment (orangegzx) [inline content/zh/docs/setup/install/operator/index.md:110]:
```suggestion
Istio Operator 控制器在创建 `IstioOperator` 资源的 90 秒内启动 Istio 的安装。
```

File: content/zh/docs/setup/install/operator/index.md [hunk 4]
```diff
@@ -224,6 +241,22 @@ $ helm install istio-operator manifests/charts/istio-operator \
...
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+  name: example-istiocontrolplane-1-8-1
+spec:
+  revision: 1-8-1
+  profile: demo
+{{< /text >}}
+
 运行该命令后，你将看到两组并排运行的控制平面 deployments 和 services：
```
Comment (orangegzx) [inline content/zh/docs/setup/install/operator/index.md:260]:
```suggestion
运行该命令后，你将看到两组并排运行的控制平面 Deployments 和 Services：
```

File: content/zh/docs/setup/install/operator/index.md [hunk 5]
```diff
@@ -248,28 +281,18 @@ istiod-1-8-1    ClusterIP   10.87.4.92   <none>        15010/TCP,15012/TCP,443/T
 如果你使用 operator 完成了控制平面的金丝雀升级，
 请运行以下命令卸载旧版本的控件平面，并保留新版本：
```
Comment (orangegzx) [inline content/zh/docs/setup/install/operator/index.md:282]:
```suggestion
如果你使用 Operator 完成了控制平面的金丝雀升级，请运行以下命令卸载旧版本的控件平面，并保留新版本：
```

File: content/zh/docs/setup/install/operator/index.md [hunk 6]
```diff
@@ -248,28 +281,18 @@ istiod-1-8-1    ClusterIP   10.87.4.92   <none>        15010/TCP,15012/TCP,443/T
...
 
-或：
-
-{{< text bash >}}
-$ kubectl delete ns istio-operator --grace-period=0 --force
-{{< /text >}}
+如果省略 `revision` 标志，则 Istio operator 的所有修订版本都将被删除。
 
 注意：在 Istio 完全移除之前删除 operator 可能会导致 Istio 资源残留。
 需要清理 operator 未删除的内容：
```
Comment (orangegzx) [inline content/zh/docs/setup/install/operator/index.md:298]:
```suggestion
注意：在 Istio 完全移除之前删除 Operator 可能会导致 Istio 资源残留。
需要清理 Operator 未删除的内容：
```

File: content/zh/docs/setup/install/operator/index.md [hunk 7]
```diff
@@ -34,20 +31,20 @@ operator 控制器将为你应用相应的配置更改。
...
-    - operator 控制器的 deployment 对象
-    - 一个用来访问 operator 指标的服务
-    - Istio operator 运行必须的 RBAC 规则
+    - Operator 自定义资源定义（CRD）
+    - Operator 控制器的 deployment 对象
+    - 一个用来访问 Operator 指标的服务
+    - Istio Operator 运行必须的 RBAC 规则
 
-    你可以配置 operator 控制器安装的命名空间、operator 观测的命名空间、Istio 的镜像源和版本、以及更多。
+    你可以配置 Operator 控制器安装的命名空间、Operator 观测的命名空间、Istio 的镜像源和版本、以及更多。
```
Comment (kebe7jun) [inline content/zh/docs/setup/install/operator/index.md:47]:
```suggestion
    您可以配置 Operator 控制器安装的命名空间、Operator 观测的命名空间、Istio 的镜像源和版本、以及更多。
```

File: content/zh/docs/setup/install/operator/index.md [hunk 8]
```diff
@@ -11,19 +11,16 @@ owner: istio/wg-environments-maintainers
 Istio [operator](https://kubernetes.io/zh/docs/concepts/extend-kubernetes/operator/)
 管理安装。
 这样做还能缓解管理不同 Istio 版本的负担。
-你只需简单的更新 operator {{<gloss CRDs>}}自定义资源（CR）{{</gloss>}}即可，
-operator 控制器将为你应用相应的配置更改。
+你只需简单的更新 Operator {{<gloss CRDs>}}自定义资源（CR）{{</gloss>}}即可，
```
Comment (kebe7jun) [inline content/zh/docs/setup/install/operator/index.md:14]:
```suggestion
您只需简单的更新 Operator {{<gloss CRDs>}}自定义资源（CR）{{</gloss>}}即可，
```

File: content/zh/docs/setup/install/operator/index.md [hunk 9]
```diff
@@ -11,19 +11,16 @@ owner: istio/wg-environments-maintainers
...
+Operator 控制器将为你应用更改的相应配置。
 
-当你用 [istioctl install 命令](/zh/docs/setup/install/istioctl)安装 Istio 时，
-底层使用的是和 operator 安装相同的
-[`IstioOperator` API](/zh/docs/reference/config/istio.operator.v1alpha1/)。
-在这两种场景下，都会以架构验证配置，并执行同样的正确性检查。
+当你用[istioctl install 说明](/zh/docs/setup/install/istioctl)安装 Istio 时，底层使用的是和 Operator 安装相同的[`IstioOperator` API](/zh/docs/reference/config/istio.operator.v1alpha1/)。在这两种场景下，都会以架构验证配置，并执行同样的正确性检查。
```
Comment (kebe7jun) [inline content/zh/docs/setup/install/operator/index.md:17]: https://istio.io/latest/zh/docs/setup/install/istioctl/
这个 title 叫 使用 Istioctl 安装

**Review by kebe7jun — COMMENTED** [review]
用'您'不要用'你'。

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**